zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2083,13 +1845,7 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
-        "500":
-          description: >
-            An internal error occurred while processing the request.
-          content:
-            application/problem+json:
-              schema:
-                $ref: "#/components/schemas/ProblemDetail"
+        "500": {$ref: "#/components/responses/InternalServerError"}
```

🔧 For consistency in the spec, can we use the "verbose" notation? If we want to move to the condensed format, we should do that holistically.
```suggestion
        "500":
          $ref: "#/components/responses/InternalServerError"
```